README.md

```diff
@@ -90,6 +94,13 @@ Below is a list of all available snippets and the triggers of each one.
 | `lock`   | sync.Mutex Lock and defer Unlock <pre>mu.Lock()<br/>defer mu.Unlock()</pre>|
 | `nb` | non-blocking channel send <pre>select {<br/>case msg &lt;- msgChan:<br/>default:<br/>}</pre>|
 
+### Testify Assert
+| Prefix  | Content |
+| :------- | ------- |
+| `anil`   | assert nil `assert.Nil(t, actual)` |
+| `annil`   | assert not nil `assert.NotNil(t, actual)` |
+| `aeq`   | assert nil `assert.Equal(t, expected, actual)` |
+| `anerr`   | assert nil `assert.NoError(t, err)` |
 
```

Comment:
Can you add a `Import` section and put in the `Logrus import` snippet?

Reply:
Sure, I've already added those